_includes/scripts.html

```diff
@@ -8,6 +8,9 @@
 <script src="/assets/js/{{ js_file }}.js" type="text/javascript"></script>
 {% endfor %} {% endif %} {% if site.gtm_id %}
 
+<!-- HubSpot Embed Code per https://knowledge.hubspot.com/reports/install-the-hubspot-tracking-code -->
+<script type="text/javascript" id="hs-script-loader" async defer src="//js.hs-scripts.com/8376079.js"></script>
```

Comment:
Out of curiosity, is 8376079 our customer ID? Or does HubSpot figure it out from the domain?

Reply:
Yeah, that's our hubspot customer ID. It's actually included in all our Hubspot URLs.